code/modules/mob/living/carbon/xenomorph/damage_procs.dm

```diff
@@ -12,21 +12,27 @@
 		if(1)
 			switch(xeno_explosion_resistance)
 				if(3)
-					apply_damage(rand(125, 175), BRUTE) //Used to be rand(200, 300) which would one-shot most things. Tanks have severity 1 explosions on direct hits...
-					updatehealth()
+					b_loss += rand(70, 80)
+					f_loss += rand(70, 80)
+					adjust_stagger(1.1)
+					add_slowdown(1.1)
```

Comment:
Same for slowdown, it's an integer paired with a `0.3` define. This would increase it in `0.33`, to little effect.